include/verilated_funcs.h

```diff
@@ -549,23 +559,36 @@ static inline QData VL_EXTENDS_QQ(int, int lbits, QData lhs) VL_PURE {
 static inline WDataOutP VL_EXTENDS_WI(int obits, int lbits, WDataOutP owp, IData ld) VL_MT_SAFE {
     const EData sign = VL_SIGNONES_E(lbits, static_cast<EData>(ld));
     owp[0] = ld | (sign & ~VL_MASK_E(lbits));
-    for (int i = 1; i < VL_WORDS_I(obits); ++i) owp[i] = sign;
+    if (sign) {
```

Comment:
Push owp[0] into the if branches, then can remove sign as a variable.   (It was otherwise before to avoid the branch, but now it's there....)

Similar below.

With those feel free to squash merge once passes.

Reply:
Note also you can use just "if (VL_SIGN_I(lbits, ld))" to save an instruction vs VL_SIGNONES_E.  Though compiler might figure that out itself.